style.css

```diff
@@ -0,0 +1,242 @@
+*{
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+input, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5838315)